packages/app/components/Queue/UpdateQuestionPopover.tsx

```diff
@@ -0,0 +1,23 @@
+import { Popconfirm } from "antd";
+import { ReactElement } from "react";
+
+interface UpdateQuestionPopoverProps {
+  onConfirm: () => void;
+  children: ReactElement;
+}
+
+export default function UpdateQuestionPopover({
+  onConfirm,
+  children,
+}: UpdateQuestionPopoverProps): ReactElement {
+  return (
+    <Popconfirm
+      title="Are you sure you want to delete this question from the queue?"
```

Comment:
Should this title be passed in as `props` so it can be set dynamically for the use case?

When a TA clicks the can't find button, it could be slightly confusing to see the word "delete" in the title of the Popover. Maybe we could say something like 'Are you sure you want to mark this question as "Can't Find"?'

Reply:
sure... might just remove `UpdateQuestionPopover` and then just use normal Popconfirm at that point cause that's a little too much dryness or w/e

Reply:
Cool, that sounds good